boards/renesas/da1469x_dk_pro/da1469x_dk_pro-pinctrl.dtsi

```diff
@@ -24,6 +24,15 @@
 		};
 	};
 
+	/omit-if-no-ref/ i2c_sleep: i2c_sleep {
+		group1 {
+			pinmux = <SMARTBOND_PINMUX(GPIO, 0, 31)>,
+				<SMARTBOND_PINMUX(GPIO, 0, 30)>;
+			bias-pull-down;
```

Comment:
I don't think should put pull-down during sleep

Reply:
Correct. Apart from leakage issues (when external pull ups are used) there is a risk to erroneously issue a start condition if the SDA line is first pulled down while the SCL line is idle.